osu.Game/Screens/Play/Player.cs

```diff
@@ -139,7 +139,7 @@ private void load(AudioManager audio, APIAccess api, OsuConfigManager config)
             adjustableClock = new DecoupleableInterpolatingFramedClock { IsCoupled = false };
 
             adjustableClock.Seek(AllowLeadIn
-                ? Math.Min(RulesetContainer.GameplayStartTime, beatmap.HitObjects.First().StartTime - beatmap.BeatmapInfo.AudioLeadIn)
+                ? Math.Min(0, beatmap.HitObjects.First().StartTime - beatmap.BeatmapInfo.AudioLeadIn)
```

Comment:
Shouldn't the second part of this be `GameplayStartTime - AudioLeadIn`?

Reply:
it really should; not sure how that got merged.